### DISCUSSION TITLE
Alexa Guard is going away

### DISCUSSION THREAD

**Discussion by lymanepp** [alandtse/alexa_media_player#2039] [General]
**_I don't know what, if any, impact this will have on the integration. I couldn't find an existing discussion or issue, so I thought I'd copy the email from Amazon here just in case._**

Dear Alexa Guard customer,   We recently announced plans to discontinue Alexa Guard and launch Alexa  Emergency Assist, a new service that helps you keep your family safe.  Some Guard features, including Smoke and CO alarm detection, will be  available as part of Emergency Assist for $5.99/month or $59/year (visit  amazon.com/alexaemergencyassist  for up to date pricing details). Other Guard features, including Away  Lighting, will now be available as part of your general Alexa  experience.   See additional details in the Q&A below. Thank you for being an Alexa customer.

**What is happening to Alexa Guard?** We are discontinuing Alexa Guard and transitioning some Guard features to a new subscription service, Alexa Emergency Assist.

**When will Alexa Guard be removed?** In the coming months. We will reach out again with additional details.

**What is Alexa Emergency Assist?** Alexa Emergency Assist is a new safety service. We’ve combined the top  features you know and love from Alexa Guard and improved them to make it  even easier for you to get 24/7 hands-free help from first responders  when you need it most. For example, you no longer need to remember to  turn on Away mode — with Alexa Emergency Assist, Alexa can detect smoke  and CO alarms and send an alert to you 24/7. We’ve also added features  like Emergency Contacts, the people who will be notified when you or  someone in your house says, "Alexa, call for help”, and Critical  Information, which allows you to pre-save important details like your  gate/door code or medical information to local police, fire, or  ambulance responders. We think these features, combined with access to  professionally-trained Urgent Response agents that can triage and  request dispatch 24/7, 365 days a year, are something that will give you  even more peace of mind. Learn more here.

**How can I access other Guard features?** This transition will impact whether and how you can access the following Alexa features: 
**Home and Away Modes**: You can continue to use Home/Away modes  for free to sync a supported security system to Alexa modes, control  Away Lighting, and trigger Routines.

**Away Lighting**: You can continue to use Away Lighting with Home/Away modes for free.

**Smoke and CO alarm sound detection**: This is only available to Emergency Assist subscribers.

**Glass break sound detection**: This will only be available to Emergency Assist subscribers starting soon.

If you've previously linked a security system to Guard and/or set  up Away Lighting, we will preserve your existing preferences so you see  no gap in functionality. You can view and update your Home/Away mode  preferences today via Guard settings.

**How does this impact Amazon Astro?** You can continue to link Astro to Alexa in the Alexa app and to use  Alexa Home/Away modes to change Astro modes. If you are an Astro  customer with a Ring Protect Pro subscription, Alexa Guard Plus was  included as a part of your Ring Protect Pro subscription. As a part of  this transition to Alexa Emergency Assist, we are removing Guard Plus as  a part of the features included in Ring Protect subscriptions. To help  ease this transition, you can enjoy Alexa Emergency Assist at no cost to  you until 10/31/2024. No action is needed on your end for this  transition to Alexa Emergency Assist.

As a Ring Protect Pro subscriber, Astro will continue to  investigate and send Smart Alerts, as it does now, when it detects glass  breaking sounds and smoke and CO alarms while Astro is set to Away  mode. However, Alexa emergency helpline will require an Alexa Emergency  Assist subscription.

© 2023 Amazon.com,  Inc. or its affiliates. All rights reserved. Amazon, Alexa, Echo,  Prime, and all related logos are trademarks of Amazon.com, Inc. or its  affiliates. Amazon.com, 410 Terry Avenue N., Seattle, WA 98109-5210. Reference: 459031490

**Comment by genehand** [comment]
Ran into this today. Trying to enable guard mode gives an error about “arming requires a code” however that’s a paid feature now so I don’t expect it to work. They do still offer home & away modes to control lighting which is all I really used. Would be cool to have that available through this integration.
